api/config/testdata/base-configs-1.yaml

```diff
@@ -115,8 +115,9 @@ FeatureToggleConfig:
       BaseURL: https://test.io/
       DashboardRepository: dashboards/merlin
       DashboardBranch: master
-      AlertRepository: alerts/merlin
+      AlertRepository: alerts_repository/merlin
```

Comment:
This change isn't technically needed but I did this to make the field (and its purpose) more easily distinguishable from the new `AlertPathPrefix` field.